updated recoveries on district data for gp_westrand file

BadisaMosesane opened the pull request:
Updated recoveries data and the daily subdistrict cases on the gp_westrand.csv file

added daily entries for West Rand Recoveries column, WR Merafong city, WR Mogale city, WR RW city and Unallocated.

shaze commented:
@BadisaMosesane Thank you for this --- this file is automatically updated with all the other GP metros from a common source file so I don't update this file separately from all the others and it will be over-written. I see you have some older data not captured. Could you summarise the differences and I will try to update the GP source file over the next day or two.

shaze commented:
@BadisaMosesane Please could you relook at your pull request. You have added quotes to the header, which is not correct and breaks the GitHub pretty viewer which makes it hard for me to review. And then if you could just give me a one sentence explanation of what you've done then I can look at it. Again thanks for your help -- much appreciated

BadisaMosesane commented:
@shaze thanks for the feedback, I will have a look at it again, I guess the csv save format (utf csv) might have caused the quotes to the header.

I have added some data on the gp_westrand file I.e. added missing entries for the recoveries column, and added daily cases for the individual subdistricts. The idea is to populate missing values in the daily sub district cases.

shaze commented:
Great -- if you could fix the header then I will look at it and import.

BadisaMosesane commented:
Alright @shaze I will change the delimiter to comma and push

vukosim commented:
@BadisaMosesane Can you update so that we can merge

BadisaMosesane commented:
Hi Vukosi,

I had updated and shared the files with Scott.

Regards

On Fri, Jul 10, 2020 at 3:50 PM Vukosi <notifications@github.com> wrote:

> @BadisaMosesane <https://github.com/BadisaMosesane> Can you update so
> that we can merge
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dsfsi/covid19za/pull/453#issuecomment-656687161>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD3YXIE7XRSMPHHURMSKYU3R24MCRANCNFSM4N6GNEIA>
> .
>
-- 
Badisa Mosesane
Computer Science Student
University of Botswana
'Aspiring Data Scientist'

shaze commented:
Sorry -- I am the bottleneck. This is historical data -- a one-off collection. I do intend to start updating this data and will integrate when I have the time to start.
